libs/sdk-core/src/breez_services.rs

```diff
@@ -584,12 +585,13 @@ impl BreezServices {
     ) -> Result<ReverseSwapInfo> {
         match self.in_progress_reverse_swaps().await?.is_empty() {
             true => {
+                let reverse_routing_node = self.btc_send_swapper.reverse_swapper_api.fetch_reverse_routing_node().await?;
```

Comment:
If we already have the reverse_swapper_api inside the btc_send_swapper then we should use it internally to get the reverse routing node rather then fetch it at the BreezServices level.

Reply:
Done in 88946bc63ca7c7b6510fb405d1d8864fbbcfa561